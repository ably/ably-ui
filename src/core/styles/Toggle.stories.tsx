import React from "react";

export default {
  title: "CSS/Toggle",
};

export const Default = {
  render: () => (
    <label className="ui-toggle">
      <input type="checkbox" />
      <span className="ui-toggle-slider"></span>
    </label>
  ),
};

export const Toggled = {
  render: () => (
    <label className="ui-toggle">
      <input type="checkbox" checked />
      <span className="ui-toggle-slider"></span>
    </label>
  ),
};

export const Disabled = {
  render: () => (
    <label className="ui-toggle">
      <input type="checkbox" disabled />
      <span className="ui-toggle-slider"></span>
    </label>
  ),
};

export const CustomisedSliderColor = {
  render: () => (
    <label className="ui-toggle">
      <input type="checkbox" />
      <span className="ui-toggle-slider bg-neutral-800"></span>
    </label>
  ),
};
