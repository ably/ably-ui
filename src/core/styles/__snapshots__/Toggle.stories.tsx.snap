// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSS/Toggle CustomisedSliderColor smoke-test 1`] = `
<label class="ui-toggle">
  <input type="checkbox">
  <span class="ui-toggle-slider bg-neutral-800">
  </span>
</label>
`;

exports[`CSS/Toggle Default smoke-test 1`] = `
<label class="ui-toggle">
  <input type="checkbox">
  <span class="ui-toggle-slider">
  </span>
</label>
`;

exports[`CSS/Toggle Disabled smoke-test 1`] = `
<label class="ui-toggle">
  <input type="checkbox"
         disabled
  >
  <span class="ui-toggle-slider">
  </span>
</label>
`;

exports[`CSS/Toggle Toggled smoke-test 1`] = `
<label class="ui-toggle">
  <input type="checkbox"
         checked
  >
  <span class="ui-toggle-slider">
  </span>
</label>
`;
